src/extras/curves/CatmullRomCurve3.d.ts

```diff
@@ -43,6 +43,6 @@ export class CatmullRomCurve3 extends Curve<Vector3> {
 
 	points: Vector3[];
 
-	getPoint( t: number ): Vector3;
+	getPoint( t: number, optionalTarget?: Vector3 ): Vector3;
```

Comment:
How about deleting the entire `getPoint()` definition? `CatmullRomCurve3` is derived from `Curve` where `getPoint()` is already defined.

Reply:
Yeah, that would also work. I assumed there was a reason why someone redefined this method, so I didn't feel comfortable just deleting it. But if you think that's best, I can remove the line altogether.

Reply:
I think the definition is redundant. This is also true for `CubicBezierCurve3`,  `LineCurve3` and `QuadraticBezierCurve3`. Not sure why these methods are still there.

Reply:
Ok, I can modify those files tonight. Stay tuned...